src/pages/accomodation/components/Reviews.tsx

```diff
@@ -0,0 +1,118 @@
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import KeyboardArrowLeftRoundedIcon from '@mui/icons-material/KeyboardArrowLeftRounded';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import Rating from '@mui/material/Rating';
+import Box from '@mui/material/Box';
+
+import useTheme from '@mui/material/styles/useTheme';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useCallback, useState } from 'react';
+
+import { Review } from './Review';
+import useGetReviewsQuery from '../../../api/queries/reviews/useGetReviewsQuery';
+import LoadingPrimary from '../../../components/loader/LoadingPrimary';
+
+export const Reviews = ({ accommodationId }: { accommodationId: string }) => {
+  const { data: reviews, isError, isLoading } = useGetReviewsQuery(accommodationId);
+  const [open, setOpen] = useState(false);
+
+  const theme = useTheme();
+  const mobileScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+  const toggleModal = useCallback(() => {
+    setOpen((prev) => !prev);
+  }, []);
+
+  if (isLoading || isError) return <LoadingPrimary />;
```

Comment:
I think it's better to should "No items" screen instead of loading screen in case of error